mhs/inbound/main.py

```diff
@@ -85,10 +85,20 @@ def start_inbound_server(local_certs_file: str, ca_certs_file: str, key_file: st
     healthcheck_application = tornado.web.Application([
         ("/healthcheck", healthcheck_handler.HealthcheckHandler)
     ])
-    healthcheck_application.listen(80)
+    server_port = int(config.get_config('INBOUND_SERVER_PORT', default='80'))
+    healthcheck_application.listen(server_port)
```

Comment:
This port is used just for healthcheck - all operations are done through 443 (see line 83). Should the other one be configurable too?

Reply:
Good catch! I think all of the services have two ports - healthcheck and API.

Reply:
AFAIK the rest are using just one port

Reply:
I assume we shouldn't change that 443 even for local development as it will break spine unsolicited responses I guess

anyway the main reason was to change the port 80 to something else as all 3 were using it
443 looks to be used by inbound only

Reply:
So `INBOUND_SERVER_PORT` is a little misleading. Maybe `INBOUND_HEALTHCHECK_SERVER_PORT`?

Might be good to change the variable name also